.changeset/gold-mice-pretend.md

```diff
@@ -0,0 +1,6 @@
+---
+"remix": patch
+"@remix-run/dev": patch
+---
+
+use esbuild's new automatic jsx transform
```

Comment:
Can we add a little context here? What does this actually solve for the user since there's no difference in usage from our perspective?

Reply:
done! let me know if there's anything that should be noted :)